docs/Podman_instructions.md

```diff
@@ -44,4 +46,8 @@ is running.
 `podman exec -it synchweb-dev /bin/bash`.  From here, check the required
 php and httpd processes are running and check the logs under
 `/var/log/httpd`.
+* Sometimes the web client stops working.  Not sure why, but this can usually
```

Comment:
@morybc for development pueposes, can we not run `yarn run serve` to have the client rebuilt when we make changes?